packages/agreement-process/src/model/domain/toEvent.ts

```diff
@@ -21,59 +15,6 @@ import {
 } from "pagopa-interop-models";
 import { match } from "ts-pattern";
 
-export const toAgreementStateV1 = (state: AgreementState): AgreementStateV1 =>
```

Comment:
Instead of removing protobuf converters to V1 I would move them into a dedicated file somewhere in shared packages or `agreement-readmodel-writer` tests, because they will be used when writing V1 events tests in the readmodel-writer as done for catalog in #278 for example